dapr-spring/dapr-spring-boot-starters/dapr-spring-boot-starter/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

```diff
@@ -0,0 +1,3 @@
+io.dapr.spring.boot.autoconfigure.client.DaprClientAutoConfiguration
+io.dapr.spring.boot.autoconfigure.pubsub.DaprPubSubAutoConfiguration
```

Comment:
@artur-ciocanu ok.. so you meant this.. but I strongly thing that we don't need `io.dapr.spring.boot.autoconfigure.pubsub.DaprPubSubAutoConfiguration` and `io.dapr.spring.boot.autoconfigure.statestore.DaprStateStoreAutoConfiguration`, as I pointed in the docs for spring-data-redis and spring-kafka they ask the users to provide these @Beans in their applications. I think we should do that instead of adding our own AutoConfigurations.

Reply:
This has been addressed.